libvast/src/system/accountant.cpp

```diff
@@ -109,7 +109,12 @@ void accountant_state::command_line_heartbeat() {
   if (logger && logger->verbosity() >= CAF_LOG_LEVEL_INFO
       && accumulator.node.events > 0) {
     std::ostringstream oss;
-    oss.imbue(std::locale(""));
+    try {
+      oss.imbue(std::locale(""));
+    } catch (const std::exception& e) {
+      VAST_DEBUG_ANON("accoutant failed to set locale the statistics output:",
```

Comment:
Why are you using `VAST_DEBUG_ANON` here? `VAST_DEBUG(self, ...)` should be fine in this context. Also the sentence is off. Did you mean "... to set the locale for statistics ..."?

Reply:
The state type didn't have a `self` member. That is rectified with the follow up commits.